src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionLiteral.cs

```diff
@@ -225,7 +227,7 @@ private BoundExpression MakeCollectionLiteralSpreadElement(BoundCollectionLitera
             return _factory.SpillSequence(
                 ImmutableArray<LocalSymbol>.Empty,
                 ImmutableArray.Create(statement),
-                result: _factory.Literal(0)); // PROTOTYPE: The result is unused. Can we avoid creating it entirely?
+                result: _factory.Literal(0)); // result is unused
```

Comment:
does a later state of lowering/emit simply drop this value? Just curious.

Reply:
That is, prevent the final generated IL from including a needless "push 0; pop;" or similar.